.editorconfig

```diff
@@ -15,4 +15,6 @@ ij_kotlin_allow_trailing_comma = false
 ij_kotlin_allow_trailing_comma_on_call_site = false
 
 ktlint_standard_argument-list-wrapping = disabled
-ktlint_function_naming_ignore_when_annotated_with = Composable
+ktlint_standard_class-signature = disabled
+ktlint_standard_function-expression-body = disabled
```

Comment:
these got enabled because of their move from experimental to standard, right?

Reply:
exactly!